### PR TITLE
Changed sampling time for heatmap from hours to minutes

### PR DESCRIPTION
**Pull request by RobertoCastellini87**
Needed for new component d1-heatmap - still not merged.

### CODE REVIEW

**Comment by RobertoCastellini87** [comment]
@antoineDeRengerve Can you check and merge?

File: src/services/ieq/ieq.js [hunk 1]
```diff
@@ -398,7 +398,7 @@ IEQ.prototype.getHeatMapData = function(sensorNames,time, callback){
 	//var endParse = new Date(endEpoch);
 	var dataConfig = {
 		criteria: {
-			time: {start: time.startEpoch, end: time.endEpoch, sampling: 'hour'}, // 360h -> 15d // 180h -> 7j
+			time: {start: time.startEpoch, end: time.endEpoch, sampling: 'minute'}, // 360h -> 15d // 180h -> 7j
```
Comment (antoineDeRengerve) [inline src/services/ieq/ieq.js]: remove useless comment

**Comment by antoineDeRengerve** [comment]
Please remove unecessary commented code + incorrect comments

**Comment by RobertoCastellini87** [comment]
@antoineDeRengerve Removed unnecessary comments

**Comment by antoineDeRengerve** [comment]
@stephanieder Merge is done!

**Pull request merged**